docker/README.md

```diff
@@ -0,0 +1,79 @@
+# Docker Compose files
+
+## docker directory
+- docker-compose.yml
+  - Starts DSpace Angular with Docker Compose from the current branch.  This file assumes that a DSpace 7 REST instance will also be started in Docker.
+- docker-compose-rest.yml
+  - Runs a published instance of the DSpace 7 REST API - persists data in Docker volumes
+- docker-compose-travis.yml
+  - Runs a published instance of the DSpace 7 REST API for CI testing.  The database is re-populated from a SQL dump on each startup.
+- cli.yml
+  - Docker compose file that provides a DSpace CLI container to work with a running DSpace REST container.
+- cli.assetstore.yml
+  - Docker compose file that will download and install data into a DSpace REST assetstore.  This script points to a default dataset that will be utilized for CI testing.
+- environment.dev.js
+  - Environment file for running DSpace Angular in Docker
+- local.cfg
+  - Environment file for running the DSpace 7 REST API in Docker.
+
+
+## To refresh / pull DSpace images from Dockerhub
+```
+docker-compose -f docker/docker-compose.yml pull
+```
+
+## To build DSpace images using code in your branch
+```
+docker-compose-f docker/docker-compose.yml build

```

Comment:
Small typo here.  Needs a space between `docker-compose` and `-f`

Reply:
@tdonohue, I originally had docker-compose.yml in the root directory.  The challenge is that the volume mounts for environment.dev.js and local.cfg become inconsistent based on the location of the first docker-compose.yml file that you reference.
In my testing, I was experiencing some unexpected behaviors because the system was sometimes starting with a valid environment.dev.js and sometimes attempting to create an empty one.

After I put all of the compose files into the same folder, I was able to complete my tests successfully.